service/controller/internal/cloudconfig/template/systemd-networkd-eth1-network.go

```diff
@@ -1,18 +1,9 @@
 package template
 
 const SystemdNetworkdEth1Network = `
-# ensure that traffic arriving on eth1 leaves again from eth1 to prevent asymetric routing
+# disable DHCP
 [Match]
 Name=eth1
 [Network]
-Address={{.MasterENIAddress}}/{{.MasterENISubnetSize}}
-
-[RoutingPolicyRule]
-Table=2
-From={{.MasterENIAddress}}/32
-
-[Route]
-Destination=0.0.0.0/0
-Gateway={{.MasterENIGateway}}
-Table=2
+DHCP=no
```

Comment:
this configuration has moved to this tool https://github.com/giantswarm/aws-attach-etcd-dep/pull/20

Now we just make sure that the interface doesn't use DHCP when it's attached